src/Sulu/Bundle/MediaBundle/Media/FormatOptions/FormatOptionsManager.php

```diff
@@ -121,8 +121,8 @@ public function save($mediaId, $formatKey, array $data)
         if (!isset($formatOptions)) {
             $formatOptions = new FormatOptions();
             $formatOptions->setFileVersion($fileVersion);
-            $fileVersion->addFormatOptions($formatOptions);
             $formatOptions->setFormatKey($formatKey);
+            $fileVersion->addFormatOptions($formatOptions);
```

Comment:
This change is not necessary as adding can be done before it. In PHP a object is based by reference and not a copy so you can edit it.

Reply:
Ah I see what you mean, ignore my comment :)